### ISSUE TITLE
error on trying to hide or unhide cell

### ISSUE THREAD

**Issue by klaff** [fonsp/Pluto.jl#217]
I can create a new Pluto notebook, put something (doesn't seem to matter what) like `1+2` in a cell, then click the Show/hide code eye icon, and the following results:

```
julia> using Pluto

julia> Pluto.run()
Go to http://localhost:1234/ to start writing ~ have fun!

Press Ctrl+C in this terminal to stop Pluto

┌ Warning: Response function to message of type foldcell failed
└ @ Pluto C:\Users\klaff\.julia\packages\Pluto\FPItk\src\webserver\WebServer.jl:283
┌ Warning: Reading WebSocket client stream failed for unknown reason:
│   exception =
│    type ServerSession has no field notebook
│    Stacktrace:
│     [1] getproperty(::Pluto.ServerSession, ::Symbol) at .\Base.jl:33
│     [2] (::Pluto.var"#125#127")(::Pluto.ServerSession, ::Dict{String,Any}, ::Notebook, ::Cell; initiator::Pluto.Initiator) at C:\Users\klaff\.julia\packages\Pluto\FPItk\src\webserver\Dynamic.jl:126
│     [3] process_ws_message(::Pluto.ServerSession, ::Dict{String,Any}, ::HTTP.WebSockets.WebSocket{HTTP.ConnectionPool.Transaction{Sockets.TCPSocket}}) at C:\Users\klaff\.julia\packages\Pluto\FPItk\src\webserver\WebServer.jl:281
│     [4] (::Pluto.var"#71#75"{Pluto.ServerSession})(::HTTP.WebSockets.WebSocket{HTTP.ConnectionPool.Transaction{Sockets.TCPSocket}}) at C:\Users\klaff\.julia\packages\Pluto\FPItk\src\webserver\WebServer.jl:143
│     [5] upgrade(::Pluto.var"#71#75"{Pluto.ServerSession}, ::HTTP.Streams.Stream{HTTP.Messages.Request,HTTP.ConnectionPool.Transaction{Sockets.TCPSocket}}; binary::Bool) at C:\Users\klaff\.julia\packages\HTTP\ZggHU\src\WebSockets.jl:146
│     [6] upgrade at C:\Users\klaff\.julia\packages\HTTP\ZggHU\src\WebSockets.jl:129 [inlined]
│     [7] (::Pluto.var"#70#74"{Pluto.ServerSession,HTTP.Handlers.Router{Symbol("##255")}})(::HTTP.Streams.Stream{HTTP.Messages.Request,HTTP.ConnectionPool.Transaction{Sockets.TCPSocket}}) at C:\Users\klaff\.julia\packages\Pluto\FPItk\src\webserver\WebServer.jl:119
│     [8] handle at C:\Users\klaff\.julia\packages\HTTP\ZggHU\src\Handlers.jl:269 [inlined]
│     [9] #4 at C:\Users\klaff\.julia\packages\HTTP\ZggHU\src\Handlers.jl:345 [inlined]
│     [10] macro expansion at C:\Users\klaff\.julia\packages\HTTP\ZggHU\src\Servers.jl:367 [inlined]
│     [11] (::HTTP.Servers.var"#13#14"{HTTP.Handlers.var"#4#5"{HTTP.Handlers.StreamHandlerFunction{Pluto.var"#70#74"{Pluto.ServerSession,HTTP.Handlers.Router{Symbol("##255")}}}},HTTP.ConnectionPool.Transaction{Sockets.TCPSocket},HTTP.Streams.Stream{HTTP.Messages.Request,HTTP.ConnectionPool.Transaction{Sockets.TCPSocket}}})() at .\task.jl:356
└ @ Pluto C:\Users\klaff\.julia\packages\Pluto\FPItk\src\webserver\WebServer.jl:155


julia> versioninfo()
Julia Version 1.5.0-rc1.0
Commit 24f033c951 (2020-06-26 20:13 UTC)
Platform Info:
  OS: Windows (x86_64-w64-mingw32)
  CPU: Intel(R) Core(TM) i7-8650U CPU @ 1.90GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-9.0.1 (ORCJIT, skylake)
Environment:
  JULIA_NUM_THREADS = 8

(@v1.5) pkg> status
Status `C:\Users\klaff\.julia\environments\v1.5\Project.toml`
  [c52e3926] Atom v0.12.16
  [6e4b80f9] BenchmarkTools v0.5.0
  [336ed68f] CSV v0.7.3
  [9961bab8] Cbc v0.7.0
  [9e3b23bb] Cloudy v0.1.0
  [a93c6f00] DataFrames v0.21.4
  [459566f4] DiffEqCallbacks v2.13.4
  [0c46a032] DifferentialEquations v6.15.0
  [5789e2e9] FileIO v1.3.0
  [28b8d3ca] GR v0.51.0
  [82e4d734] ImageIO v0.3.0
  [916415d5] Images v0.22.4
  [c601a237] Interact v0.10.3
  [4076af6c] JuMP v0.21.3
  [e5e0dc1b] Juno v0.8.2
  [bdcacae8] LoopVectorization v0.8.17
  [47be7bcc] ORCA v0.4.0
  [14b8a8f1] PkgTemplates v0.7.6
  [ccf2f8ad] PlotThemes v2.0.0
  [f0f68f2c] PlotlyJS v0.13.1
  [91a5bcdd] Plots v1.5.5
  [c3e4b0f8] Pluto v0.10.7
  [f27b6e38] Polynomials v1.1.3
  [27ebfcd6] Primes v0.5.0
  [d330b81b] PyPlot v2.9.0
  [f2c3362d] RecursiveFactorization v0.1.4
  [f2b01f46] Roots v1.0.3
  [f3b207a7] StatsPlots v0.14.6
  [112f6efa] VegaLite v2.2.0
```

**Comment by klaff** [comment]
I tried reproducing this on a second Windows 10 machine, same Julia and Pluto versions and I can't get it to happen.

On the first (failing) machine, it happens in both Chrome and Firefox.

**Comment by klaff** [comment]
This may have been fixed by v0.10.8.

**Comment by fonsp** [comment]
Sorry! It was fixed in `v0.10.8` (https://github.com/fonsp/Pluto.jl/issues/216).
